### PR TITLE
dont show data from west virginia

### PR DESCRIPTION
**Pull request by joshzarrabi**
- the state has fallen below our 5% treshold of cases compared to
nytimes

### CODE REVIEW

File: frontend/src/reports/ReportProvider.tsx [hunk 1]
```diff
@@ -190,10 +191,10 @@ function ReportProvider(props: { madLib: MadLib; setMadLib: Function }) {
           Until then, the following states appear grey when viewing COVID-19
           maps featuring hospitalizations and deaths: <b>Hawaii</b>,{" "}
           <b>Nebraska</b>, <b>South Dakota</b>, and <b>Wyoming</b>.{" "}
-          <b>Delaware</b> and <b>West Virginia</b> are included when viewing
-          hospitalizations but appear as grey when viewing reports on deaths.{" "}
-          <b>Rhode Island</b> appears as grey when viewing reports on
-          hospitalizations but is included when viewing deaths.
+          <b>Delaware</b>. Is included when viewing hospitalizations but appear
```
Comment (benhammondmusic) [inline frontend/src/reports/ReportProvider.tsx:194]: remove the `.` and lowercase `Is` i think?

Comment (benhammondmusic) [reply frontend/src/reports/ReportProvider.tsx:194]: and should be `appears` now since it's a single state

Comment (joshzarrabi) [reply frontend/src/reports/ReportProvider.tsx]: thx

Comment (benhammondmusic) [reply frontend/src/reports/ReportProvider.tsx]: I'd keep the sentence structure consistent I think it should be:

```
<b>Delaware</b> is included when viewing hospitalizations, but appears as grey when viewing reports on deaths.
<b>Rhode Island</b> is included when viewing deaths, but appears as grey when viewing reports on hospitalizations.

Also, I feel like all of these `appears as grey` should be `appears grey`.... right?

File: frontend/src/reports/ReportProvider.tsx [hunk 2]
```diff
@@ -190,10 +191,10 @@ function ReportProvider(props: { madLib: MadLib; setMadLib: Function }) {
           Until then, the following states appear grey when viewing COVID-19
           maps featuring hospitalizations and deaths: <b>Hawaii</b>,{" "}
           <b>Nebraska</b>, <b>South Dakota</b>, and <b>Wyoming</b>.{" "}
-          <b>Delaware</b> and <b>West Virginia</b> are included when viewing
-          hospitalizations but appear as grey when viewing reports on deaths.{" "}
-          <b>Rhode Island</b> appears as grey when viewing reports on
-          hospitalizations but is included when viewing deaths.
+          <b>Delaware</b> is included when viewing hospitalizations but appear
```
Comment (benhammondmusic) [inline frontend/src/reports/ReportProvider.tsx:194]: still need an "s" on `appear`

Comment (joshzarrabi) [reply frontend/src/reports/ReportProvider.tsx:194]: omg

**Review by benhammondmusic — APPROVED** [review]

File: frontend/src/reports/ReportProvider.tsx (hunk 1 above)
Comment (joshzarrabi) [reply frontend/src/reports/ReportProvider.tsx]: I guess, I don't know

**Pull request merged**